src/utils.ts

```diff
@@ -160,7 +160,14 @@ const parseNumber = (value: string | number): number => {
     return isNaN(floatValue) ? 0 : floatValue;
 };
 
-const parseSettingsString = (settingsString: string): Dictionary<string>[] => {
+interface SettingMappingElement {
```

Comment:
Element is a very vague term and doesn't add any value to defining what it is

Maybe the following?

```suggestion
interface EventToPlacementKeyMapping {
```